h2/stream.py

```diff
@@ -701,6 +701,22 @@ def __init__(self,
         # The configuration for this stream.
         self.config = config
 
+    def __repr__(self):
+        stream_state_strs = {
+            StreamState.IDLE: "Idle",
+            StreamState.RESERVED_REMOTE: "Reserved Remote",
+            StreamState.RESERVED_LOCAL: "Reserved Local",
+            StreamState.OPEN: "Open",
+            StreamState.HALF_CLOSED_REMOTE: "Half Closed Remote",
+            StreamState.HALF_CLOSED_LOCAL: "Half Closed Local",
+            StreamState.CLOSED: "Closed"
+        }
```

Comment:
I'm pretty nervous about this chunk of code here, because it's fundamentally not tested in any sense. In particular, it is at risk of misbehaving if new stream states are added. Does it add much value?